src/components/events/EventForm.jsx

```diff
@@ -19,9 +20,11 @@ class EventForm extends React.Component {
 
     this.state = {
       disabledSave: true,
-      eventDate: currentValues.eventDate || '',
+      //eventDate: currentValues.eventDate || (new Date()).toLocaleDateString('en-US'),
```

Comment:
why is this commented out?

Reply:
Sorry, forgot to delete the old line of code